v-next/hardhat-errors/src/descriptors.ts

```diff
@@ -397,11 +397,11 @@ Please add the property "type" with the value "module" in your package.json to e
         websiteTitle: "Template not found",
         websiteDescription: `The template you provided is not found. Please check the documentation to learn which templates are available.`,
       },
-      WORKSPACE_NOT_FOUND: {
```

Comment:
@ChristopherDedominici Just one more q, it's OK to change this because we're still in alpha, right?